controller/handler_edge_ctrl/extend_enrollment.go

```diff
@@ -113,7 +121,10 @@ func (h *extendEnrollmentHandler) HandleReceive(msg *channel2.Message, ch channe
 
 			msg := channel2.NewMessage(env.EnrollmentCertsResponseType, body)
 
-			ch.Send(msg)
+			if err := ch.Send(msg); err != nil {
+				pfxlog.Logger().Errorf("request to extend the enrollment for a router [%s - %s] failed: %v", routerId, routerName, err)
```

Comment:
Trying to move more things to fields to make log parsing easier. It's going to be an ongoing effort :)